### PR TITLE
Cached gems

### PR DESCRIPTION
**Pull request by jage**
It's now possible to install gems without contacting the source (https://rubygems.org) with `bundle install --local`.

http://bundler.io/v1.12/bundle_package.html

### CODE REVIEW

**Comment by dentarg** [comment]
Would be great to add some notes in the README too.

**Comment by jage** [comment]
> Would be great to add some notes in the README too.

Done in c1d1686c67125e341b1476a90d6f8f0f61d84ab2

**Comment by dentarg** [comment]
From http://bundler.io/v1.12/bundle_package.html

> This will be the default on Bundler 2.0.

Do they mean that "vendoring" will be the default in Bundler 2.0?

**Comment by dentarg** [comment]
LGTM 👍

**Comment by jage** [comment]
> Do they mean that "vendoring" will be the default in Bundler 2.0?

Nah, I think it's `--all` that's the default in Bundler 2.0.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Install prerequisites
     brew install postgresql
     bundle install
 
+Ruby gems are vendored into `vendor/cache`, you should always check in the gems when changing gems. The caching is set up with [`bundle package --all`](http://bundler.io/bundle_package.html).
```
Comment (dentarg) [inline README.md]: The docs says `--all` is remembered. I guess it is, by creating/updating `.bundle/config`. But for it to be true, for all members of the project, don't we need to commit `.bundle/config` then?

**Pull request merged**